ts/formly/inputs/api-select.directive.ts

```diff
@@ -20,7 +21,10 @@ module lui.apiselect {
 			return directive;
 		}
 
-		public link(scope: IApiSelectScope, element: angular.IAugmentedJQuery): void {
+		public link(scope: IApiSelectScope, element: angular.IAugmentedJQuery, attrs: angular.IAttributes & {allowClear: boolean}): void {
+			let allowClear = attrs.allowClear;

```

Comment:
:( - just pass `scope.allowClear` to the ui-select

Reply:
but if allowClear is not set, which is the case  for each property that is not mandatory, then  allowClear will be falsy and it will cause a bug.  Therefore we want to set it to true if not defined and avoid any breaking changes

Reply:
i dont care about the breaking change, its very minor and only used by timmi and directory, so break away